monai/apps/manifest/config_parser.py

```diff
@@ -176,28 +185,35 @@ def _do_parse(self, config, id: str = ""):
         else:
             self.reference_resolver.add_item(ConfigItem(config=item_conf, id=id))
 
-    def parse(self):
+    def parse(self, reset: bool = False):
```

Comment:
added an option to do forced reset in `parse`

Reply:
I changed the default value of this arg to `True`, but actually I really didn't find a typical use case for `reset = False`, could you please help share some more info?

Thanks in advance.